debian/bullseye/foreman-release/control

```diff
@@ -0,0 +1,9 @@
+Source: foreman-release
+Build-Depends: debhelper (>= 9.20160709), rename
+Homepage: http://theforeman.org/projects/foreman
+
+Package: foreman-release
+Architecture: all
+Description: Configures the Foreman and Foreman Plugins repositories on the system. 
+The foreman-release package configures the version specific
+Foreman and Foreman Plugins repositories on the system.
```

Comment:
```suggestion
 The foreman-release package configures the version specific
 Foreman and Foreman Plugins repositories on the system.
```

Description needs to be "one space indented" so that the parser knows it still belongs to the field. (:roll_eyes:)